src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/clinicaltrialparticipation/ClinicalTrialParticipationCompositionConverter.java

```diff
@@ -0,0 +1,34 @@
+package org.ehrbase.fhirbridge.ehr.converter.specific.clinicaltrialparticipation;
+
+import org.ehrbase.fhirbridge.ehr.converter.ConversionException;
+import org.ehrbase.fhirbridge.ehr.converter.generic.ObservationToCompositionConverter;
+import org.ehrbase.fhirbridge.ehr.opt.geccostudienteilnahmecomposition.GECCOStudienteilnahmeComposition;
+import org.ehrbase.fhirbridge.ehr.opt.geccostudienteilnahmecomposition.definition.StatusDefiningCode;
+import org.hl7.fhir.r4.model.Observation;
+import org.springframework.lang.NonNull;
+
+public class ClinicalTrialParticipationCompositionConverter extends ObservationToCompositionConverter<GECCOStudienteilnahmeComposition> {
+
+    @Override
+    protected GECCOStudienteilnahmeComposition convertInternal(@NonNull Observation resource) {
+        GECCOStudienteilnahmeComposition composition = new GECCOStudienteilnahmeComposition();
+        mapStatus(composition, resource);
+        composition.setGeccoStudienteilnahme(new ClinicalTrialParticipationEvaluationConverter().convert(resource));
+        return composition;
+    }
+
+    private void mapStatus(GECCOStudienteilnahmeComposition composition, Observation obs) {
+        String status = obs.getStatusElement().getCode();
+        if (status.equals(StatusDefiningCode.FINAL.getValue())) {
+            composition.setStatusDefiningCode(StatusDefiningCode.FINAL);
+        } else if (status.equals(StatusDefiningCode.GEAENDERT.getValue())) {
+            composition.setStatusDefiningCode(StatusDefiningCode.GEAENDERT);
+        } else if (status.equals(StatusDefiningCode.REGISTRIERT.getValue())) {
+            composition.setStatusDefiningCode(StatusDefiningCode.REGISTRIERT);
+        } else if (status.equals(StatusDefiningCode.VORLAEUFIG.getValue())) {
+            composition.setStatusDefiningCode(StatusDefiningCode.VORLAEUFIG);
+        } else {
+            throw new ConversionException("The status " + obs.getStatus().toString() + " is not valid for body height.");
```

Comment:
Body height ?
Also there is a test missing for this exception.

Reply:
Habe ich bei keinem anderen Beispiel gesehen. Falscher Status liefert "DataFormatException".
Werden diese nicht schon über den StrictErrorHandler abgefangen?
(Vermutlich kann ich die Exception sowieso löschen, da diese nie erreicht wird.)
Oder verstehe ich das falsch?

Reply:
Da ist ein schreib fehler da steht Body height im String. Das ist doch ClinicalTirialParticipation und nicht Body Height ?
 obs.getStatus().toString() + " is not valid for **body height**."
 Das sollte des weiteren eine UnprocessableEntityException sein